cmd/agent/app/integrations.go

```diff
@@ -44,6 +46,8 @@ func init() {
 	tufCmd.AddCommand(searchCmd)
 	tufCmd.AddCommand(freezeCmd)
 	tufCmd.PersistentFlags().BoolVarP(&withoutTuf, "no-tuf", "t", false, "don't use TUF repo")
+	tufCmd.PersistentFlags().BoolVarP(&inToto, "in-toto", "i", false, "enable in-toto")
+	tufCmd.PersistentFlags().BoolVarP(&tufLog, "tuf-log", "l", false, "enable TUF logging")
```

Comment:
If I understand correctly this flag ends up both enabling verbose logging on pip and logging on TUF, shouldn't we just name it the `verbose` flag (shortcut `-v`)?